content/tracing/advanced_usage/_index.md

```diff
@@ -1215,14 +1215,18 @@ Coming Soon. Reach out to [the Datadog support team][contact support] to be part
 
 ## Logging
 
-Datadog's logging APIs allow for accessing active tracing identifiers, which can be used to correlate APM traces with specific log events.
+For each generated trace, there are very likely log events written by the monitored functions and applications. 
+The purpose of this section is to explain how the correlation between traces and logs can be improved by automatically adding  a trace id in your logs and then use it in the Datadog platform to show you the exact logs correlated to the observed trace.
+
+{{< img src="/src/images/tracing/logs_in_trace.png" alt="Logs in Traces" responsive="true" style="width:70%;">}}
 
 {{< tabs >}}
 {{% tab "Java" %}}
 
-The Java tracer exposes two API calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()`.
+The idea is to leverage the MDC ([Map Diagnostic Context](https://logback.qos.ch/manual/mdc.html)) to automatically add the trace and span identifier into the logs.
+The Datadog Java tracer exposes two API calls to allow printing trace and span identifiers along with log statements, `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()`.
 
-These identifiers can be injected into application logs using MDC frameworks.
+- To inject those identifier into application logs using MDC frameworks, use the following method:
```

Comment:
Just to be clear, this is an intermediate step. This part is changed with next coming releases where it will happen out of the box with the change to the format as the only requirement.